### PR TITLE
add CSS Grid Assignment

### PR DESCRIPTION
**Pull request by reaapunzell**

### CODE REVIEW

File: code/CSS Grid Assignment/style.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#wrapper {
+    font-size: 2em;
+    gap: 1rem;
+    /* Add new properties here */
+  }
+  
+  /* Add new selectors and new CSS here */
+  #wrapper{
+    display:grid;
+    grid-template-columns:4em 10em 1/2fr;
```
Comment (burrowdown) [inline code/CSS Grid Assignment/style.css:10]: 1/2fr isn't a valid measure, if you want a half you can do 0.5fr

File: code/CSS Grid Assignment/style.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    font-size: 2em;
+    gap: 1rem;
+    /* Add new properties here */
+  }
+  
+  /* Add new selectors and new CSS here */
+  #wrapper{
+    display:grid;
+    grid-template-columns:4em 10em 1/2fr;
+    grid-template-rows:
```
Comment (burrowdown) [inline code/CSS Grid Assignment/style.css:11]: remember not to leave unused properties in!

File: code/CSS Grid Assignment/style.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    grid-template-columns:4em 10em 1/2fr;
+    grid-template-rows: 
+}
+  #header{
+    grid-column-start:1;
+    grid-column-end:4;
+  }
+
+  #left{
+    width:4em;
```
Comment (burrowdown) [inline code/CSS Grid Assignment/style.css:19]: In this case, if you want the width to take up the full column, it's not necessary (and actually causes problems) to define the width separately from the grid definition. Best to let the grid handle it.

**Review by burrowdown — CHANGES_REQUESTED** [review]
I'd like you to take another try at this one, I know you can do it!

**Comment by reaapunzell** [comment]
Hi Danny, 
Thank you for the feedback. I deleted one of your comments by mistake. I did fix the issue before but did not know it would delete it permanently. Apologies. I hope the assignment is better now. Thank you

**Review by burrowdown — APPROVED** [review]
Yes, it looks good now, thank you!

**Pull request merged**